regolith/tests/test_validators.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
Can you please move the tests files to a top-level tests/ dir, rather than embedding them in the project dir? Thanks!